cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/kyan/lo-menubar-0.1.1/.waf-1.6.2-9103917fcd1bcd62db351f7433377bc6/waflib/Tools/c_config.py", line 430, in run_c_code\n    bld.compile()\n  File "/home/kyan/lo-menubar-0.1.1/.waf-1.6.2-9103917fcd1bcd62db351f7433377bc6/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task failed (exit status 1): \n\t{task 169735084: cxxprogram test.cpp.0.o -> testprog}\n[\'/usr/bin/g++\', \'test.cpp.0.o\', \'-o\', \'/home/kyan/lo-menubar-0.1.1/build/.conf_check_3baf90b2bf424fc190449eaf9dd78eaa/testbuild/testprog\', \'-Wl,-Bstatic\', \'-Wl,-Bdynamic\', \'-L/usr/lib/libreoffice/sdk/lib\', \'-luno_sal\']\n'
